sirepo/template/template_common.py

```diff
@@ -461,6 +461,20 @@ def parse_enums(enum_schema):
     return res
 
 
+def parse_mpi_log(run_dir):
```

Comment:
nit: use this in template.synergia

Reply:
I'd prefer to make a separate issue for that once this PR is in place